middleware/auth.js

```diff
@@ -104,6 +104,7 @@ module.exports = function (params) {
           }
 
           if (config.afterCallback) {
+            session = Object.assign({}, session); // serializes session
```

Comment:
serializing the session before passing it to the `afterCallback` hook.  this will remove any of the internals of the underlying `tokenSet`

Reply:
`serializing` -> `removing non-enumerable properties`